guide/getting-started/developers.md

```diff
@@ -18,13 +18,13 @@ TDS components do not include:
 
 If you are using the [Isomorphic Starter Kit](https://github.com/telusdigital/telus-isomorphic-starter-kit), TDS is already installed and configured. You can jump straight to [using the components](#3-use-tds-components).
 
-To install the latest version:
+To install the latest version of a component:
 
 ```sh
-yarn add @telusdigital/tds
+yarn add @tds/core-component-name
```

Comment:
Can we make it more apparent that the word "core" is actually part of the package names? How about this?

TDS components follow a standard naming scheme. Core components are prefixed by the word "core".

```
yarn add @tds/core-<component-name>
```